workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/WorkerBuildData.cs

```diff
@@ -1,13 +1,21 @@
 using System;
 using System.Collections.Generic;
-using Improbable.Gdk.BuildSystem.Util;
 using UnityEditor;
 
 namespace Improbable.Gdk.BuildSystem.Configuration
 {
     public class WorkerBuildData
     {
-        private readonly WorkerPlatform workerPlatform;
+        public readonly string WorkerType;
+
+        public string PackageName => $"{WorkerType}@{BuildTargetName}";
+        
+        public string BuildScratchDirectory =>
+            PathUtils.Combine(PathUtils.BuildScratchDirectory, PackageName, ExecutableName).ToUnityPath();
+        
+        private string BuildTargetName => BuildTargetNames[buildTarget];
+        private string ExecutableName => PackageName + BuildPlatformExtensions[buildTarget];
+        
```

Comment:
Can't comment on the line - but is there a particular reason we have 32 bit Windows as a common build target?

Reply:
@zeroZshadow said that we do support it and after checking in the packages, we actually do build it (just don't download it right now for the gdk)

Reply:
Developers may want to ship a 32 bit version of the game, we shouldn't really dictate whether they can or not.

Reply:
Last time we checked for this its because a lot of chineese pc's still run windows xp